### PR TITLE
fix #76450 dood.so

### LINKED ISSUE #76450: dood.so
### Issue URL (Anti Adblock Script)
[NSFW] [https://dood.so/d/58av3rpghnt7](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fdood.so%2Fd%2F58av3rpghnt7)
### Comment
Username: `@xtprion`
> 
> --
> The information below is automatically generated. Do not delete it.
> 
> Referrer:
> https://www.facecrot.uk/
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### [NSFW] Screenshots
<details>
  <summary>Screenshot 1</summary>

  [Screenshot 1](https://reports-img.adguard.com/GKABkjf.png)
</details>
<details>
  <summary>Screenshot 2</summary>

  [Screenshot 2](https://reports-img.adguard.com/xLZ4cOS.png)
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Edge 90.0.803.0 (Chromium-based)
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Self-destructing third-party cookies (180)
DNS filtering: | server: `https://dns.adguard.com/dns-query`<br>
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> Fanboy's Social Blocking List, <br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Fanboy's Annoyances, <br/>Web Annoyances Ultralist, <br/>Adblock Warning Removal List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion, <br/>AdGuard DNS, <br/>AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> ABPindo, <br/>Icelandic ABP List, <br/>AdBlockID, <br/>Dandelion Sprout's Nordic Filters, <br/>AdGuard Russian, <br/>AdGuard Japanese
Other extensions used: | bitdefender

### PR DESCRIPTION
**Pull request by wout-codes**
#76450

### CODE REVIEW

**Comment by wout-codes** [comment]
I was not able to test scriptlet rules due to AdGuard not working properly. It just responds so slow and it it responds nothing is filtered in the website. ╮(╯ _╰ )╭

**Comment by wout-codes** [comment]
@zloyden Come to this PR. I have something to discuss.

**Comment by wout-codes** [comment]
Is it okay if I make this rule a generic one? `$$script[data-cfasync="false"][wildcard="*['push']*['shift']*decodeURIComponent*"][min-length="1000"][max-length="4000"]`?

**Comment by zloyden** [comment]
> Is it okay if I make this rule a generic one? `$$script[data-cfasync="false"][wildcard="*['push']*['shift']*decodeURIComponent*"][min-length="1000"][max-length="4000"]`?

I'd leave it at that.

**Comment by wout-codes** [comment]
> > Is it okay if I make this rule a generic one? `$$script[data-cfasync="false"][wildcard="*['push']*['shift']*decodeURIComponent*"][min-length="1000"][max-length="4000"]`?
> 
> I'd leave it at that.

What do you mean? Actually, I am tryna make the Apps better. Actually there are quite a lot of sites that uses this code and I am not able to recall the sites. Since the extension can block popups using this `$popup` or `$document,popup`, so why not make the apps also block popups.

**Comment by zloyden** [comment]
I mean, I wouldn't make this rule a generic one.

**Comment by wout-codes** [comment]
> I mean, I wouldn't make this rule a generic one.

Why though? I think it should be a generic one and I have my reasons stated above.

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -550,6 +550,10 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/76450
+dood.so$$script[tag-content="adblock"][min-length="500"][max-length="1000"]
+!+ PLATFORM(ios, ext_android_cb, ext_safari, ext_ff)
+@@||dood.so^$generichide
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:556]: `dood.*#$#.pub_300x250.pub_300x250m.pub_728x90.text-ad.textAd.text_ad.text_ads.text-ads.text-ad-links { display: block !important; }` this is working fine on my end. I'll make changes directly.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:556]: @zloyden It get overwritten on my end.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:556]: @zloyden It's Okay I don't mind. But can you make that rule a generic one?

File: EnglishFilter/sections/general_extensions.txt [hunk 2]
```diff
@@ -4145,7 +4145,7 @@ youporngay.com$$script[tag-content="{""zone_id"":"""]
...
+dood.so,sports24.*,videoproxy.*,dropgalaxy.com,hikarinoakari.com,givemenflstreams.com,givemenbastreams.com,givememmastreams.com,givemeredditstreams.com,skymovieshd.vin,proxybit.*,ytmp3eu.eu,virpe.com,ownurl.website,300mbplus.*,hdmovieplus.site,get-to.link,chiasecongnghe.com,mangovideo.club,pctmix.com,mirrorace.*,300mbfilms.*,youdbox.com,gaypornmasters.com,wordcounter.icu,darmowatv.ws,baixarhentai.net,scnlog.me,itopmusic.com,onlystream.tv,animesvision.biz,koreanaddict.net,nowagoal.xyz,bestjavs.com,thepiratebay.asia,nocensor.rest,thepiratebay0.org,frkmusic.me,onceddl.org,freeomovie.to,nxload.com,uii.io,short.pe,cryptohog.net,clicknupload.co,viitodo.com,snahp.it,hikarinoakariost.info,animeshouse.net,drobfilx.com,skymovieshd.fit,porncomix.info,vldtodo.com,xyzembed379.net,zobacz.ws,legendas.dev,shinden.pl,megasubtitles.com,haloani.ru,sukidesuost.info,ouofly.com,mixdrop.co,putload.tv,exdb.link,exdb.net,skidrowcodex.net,filmisub.com,drycounty.com,guard.link,ouo.io,shon.xyz,bitconews.net$$script[data-cfasync="false"][wildcard="*['push']*['shift']*decodeURIComponent*"][min-length="1000"][max-length="4000"]
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:4148]: `dood.*`. We can use `*` here.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/general_extensions.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:4148]: Generic/general?

File: EnglishFilter/sections/general_extensions.txt [hunk 3]
```diff
@@ -34,7 +34,7 @@ xmoviesforyou.video,streamz.cc,drycounty.com,vidz72.com,uplod.ws,pandamovie.net,
...
....com,fx4vip.com,onlystream.tv,animesvision.biz,koreanaddict.net,iklandb.com,moaplos.com,nowagoal.xyz,bestjavs.com,thepiratebay.asia,nocensor.rest,dramahd.me,thepiratebay0.org,scat.gold,aparat.cam,clik.pw,animeblkom.net,frkmusic.me,vstplugs.com,onceddl.org,freeomovie.to,leechpremium.link,nxload.com,uii.io,short.pe,cryptohog.net,abcvideo.cc,clicknupload.co,viitodo.com,intothelink.com,snahp.it,hikarinoakariost.info,streamtape.com,animeshouse.net,drobfilx.com,skymovieshd.fit,porncomix.info,7r6.com,vldtodo.com,xyzembed379.net,zobacz.ws,vidtome.stream,legendas.dev,megasubtitles.com,haloani.ru,sukidesuost.info,ouofly.com,mixdrop.co,putload.tv,exdb.link,exdb.net,animeuniverse.it,filmisub.com,drycounty.com,guard.link,mitly.us,ouo.io,shon.xyz,bitconews.net,cda-tv.pl,picbaron.com,skidrowcodex.net,streamz.*,business-degrees.me,business-mortgage.pw,business-loan.trade,credits-loan.pw,business-credits.cc,business-mortgage.info,business-loans.pw,loan-trading.net,videogreen.xyz,thisav.com,filma24.to,playtube.ws,pornxbit.com,hitomi.la,rnbxclusive.best,onceddl.com,pornxday.com,autofaucets.org,rifurl.com,filmninja.ws,imgrock.net,shurt.pw,piratebay.live#%#//scriptlet("abort-on-property-write", "_pop")
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:37]: Do you need this rule? I've checked it with app/extension and US IP. Only HTML rule requiers on my end.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:37]: Both works.

File: EnglishFilter/sections/general_extensions.txt (hunk 2 above)
Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt:4148]: I don't think that it should be a generic one. In this way the rule is more secure than if you make it a generic one. that's why I share @ameshkov opinion on this matter.

Comment (ameshkov) [reply EnglishFilter/sections/general_extensions.txt:4148]: Yep, I generally don't like generic rules, they cause too much trouble.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:4148]: Okay then.

File: EnglishFilter/sections/general_extensions.txt (hunk 3 above)
Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt:37]: Ok, let's use `dood.*` here too.

**Review by zloyden — APPROVED** [review]

**Pull request merged**